src/main.c

```diff
@@ -210,6 +210,13 @@ static gboolean install_start(int argc, char **argv)
 
 	r_exit_status = 1;
 
+#if ENABLE_SERVICE == 1
```

Comment:
Why do we only default to `/etc/rauc/system.conf` if the service enabled? It's more relevant in the other case.

Reply:
Seems as if I accidentally inverted the logic, yes.